bait-files/src/main/java/com/google/sps/servlets/RequestServlet.java

```diff
@@ -24,5 +36,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) {}
-}
\ No newline at end of file
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, String[]> parameters = request.getParameterMap();
+    Long requestId = null;
+    // will get user id from input when connected
+    // String userIdInput = parameters.get("user-id-input")[0];
+    String nameInput = parameters.get(NAME)[0];
+    String usernameInput = parameters.get(USERNAME)[0];
+    String emailInput = parameters.get(EMAIL)[0];
+    String addressInput = parameters.get(ADDRESS)[0];
+    String pictureInput = parameters.get(PICTURE)[0];
+    String phoneInput = parameters.get(PHONE)[0];
+    String notesInput = parameters.get(NOTES)[0];
+
+    // id is 1 for now by default until I can get userId from user feature
```

Comment:
Good call to write a comment explaining the "1", which would have otherwise been confusing =)